src/Sylius/Bundle/ProductBundle/Behat/ProductContext.php

```diff
@@ -13,7 +13,7 @@
 
 use Behat\Gherkin\Node\TableNode;
 use Doctrine\Common\Collections\ArrayCollection;
-use Sylius\Bundle\AttributeBundle\AttributeType\CheckboxAttributeType;
+use Sylius\Component\Attribute\AttributeType\CheckboxAttributeType;
```

Comment:
Yeah, I should mention that I also fix bad coupling between bundles/components where possible. :P In order to fix their builds.